### PR TITLE
Add check points

### PR DESCRIPTION
**Pull request by wagner-deoliveira**
Check if no value is entered at input field. 
Check if metamask is installed. 
Remove button displayed for small screens.

### CODE REVIEW

File: js/cheap.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
+window.addEventListener('load', function() {
+  if (typeof web3 !== 'undefined') {
+    console.log('web3 is enabled')
+    if (web3.currentProvider.isMetaMask === true) {
+      console.log('MetaMask is active')
+    } else {
+      alert('MetaMask is not available, please install MetaMask extension')
+    }
+  } else {
+    alert('Web3 is not found. Please install MetaMask extension')
```
Comment (pymike00) [inline js/cheap.js:10]: More a request for clarification for my own knowledge than a comment on the code: is web3 not provided by the CDN?

**Comment by pymike00** [comment]
Just a request for clarification on my side please, being this the first time I am using this stack (it might also be useful to other devs reading us)

Is the web3 object not provided already via the CDN script?  (see comment on code)
Or is it Metamask that returns an instance of it, hence we get undefined if Metamask is not installed?

Code looks good! 👍🏻 👍🏻 👍🏻 


EDIT:
wait guess I am answering myself lol, I guess we get a "web3" object from Metamask and we can use "Web3" to access utils such as "Web3.utils.toWei" ?

**Review by flyerman — APPROVED** [review]
👍

**Comment by wagner-deoliveira** [comment]
> Just a request for clarification on my side please, being this the first time I am using this stack (it might also be useful to other devs reading us)
> 
> 
> 
> Is the web3 object not provided already via the CDN script?  (see comment on code)
> 
> Or is it Metamask that returns an instance of it, hence we get undefined if Metamask is not installed?
> 
> 
> 
> Code looks good! 👍🏻 👍🏻 👍🏻 
> 
> 
> 
> 
> 
> EDIT:
> 
> wait guess I am answering myself lol, I guess we get a "web3" object from Metamask and we can use "Web3" to access utils such as "Web3.utils.toWei" ?


Actually web3 is only installed/instantiated in the browser if you install an app or extension that uses it (as far as I understand). You can easily test it by using a browser where no web3 extension, for example MetaMask, is not installed ;)

Thanks for the feedback.

**Comment by pymike00** [comment]
> > Just a request for clarification on my side please, being this the first time I am using this stack (it might also be useful to other devs reading us)
> > Is the web3 object not provided already via the CDN script?  (see comment on code)
> > Or is it Metamask that returns an instance of it, hence we get undefined if Metamask is not installed?
> > Code looks good! 👍🏻 👍🏻 👍🏻
> > EDIT:
> > wait guess I am answering myself lol, I guess we get a "web3" object from Metamask and we can use "Web3" to access utils such as "Web3.utils.toWei" ?
> 
> Actually web3 is only installed/instantiated in the browser if you install an app or extension that uses it (as far as I understand). You can easily test it by using a browser where no web3 extension, for example MetaMask, is not installed ;)
> 
> Thanks for the feedback.

Awesome, thank you for the clarification! =)

**Pull request merged**